### PR TITLE
cleanup compiler toolwindow... remove padding

### PR DESCRIPTION
**Pull request by clojj**

### CODE REVIEW

**Comment by clojj** [comment]
PR is actually just these 2 commits:
a7a63d2 
e4d76e2

**Comment by clojj** [comment]
PR consists of these commits:
a7a63d2 
e4d76e2
48d0f7c
566a6c4

**Comment by clojj** [comment]
PR consists of these commits:
a7a63d2
e4d76e2
48d0f7c
566a6c4
57c0668

**Comment by clojj** [comment]
target

**Comment by klazuka** [comment]
Thanks! I’m on vacation right now, but I will check this I’m out when I return early next week.

**Comment by clojj** [comment]
target

**Comment by clojj** [comment]
> Thanks! I’m on vacation right now, but I will check this I’m out when I return early next week.

enjoy your vacation !

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -140,7 +140,7 @@
     </change-notes>
 
     <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-    <idea-version since-build="182"/>
+    <idea-version since-build="182.3684.2"/>
```
Comment (klazuka) [inline src/main/resources/META-INF/plugin.xml]: Why is this necessary? Was I doing it wrong before?
